packages/binding-http/src/http-server.ts

```diff
@@ -430,7 +431,11 @@ export default class HttpServer implements ProtocolServer {
                 const oAuthScheme = thing.securityDefinitions[thing.security[0] as string] as OAuth2SecurityScheme;
 
                 // TODO: Support security schemes defined at affordance level
-                const scopes = oAuthScheme.scopes ?? [];
+                const scopes = oAuthScheme.scopes
```

Comment:
Actually is not nested `?` operator but it is a TS operator for [Nullish Coalescing](https://mariusschulz.com/blog/nullish-coalescing-the-operator-in-typescript). Is totally fine in TS code. I guess @danielpeintner removed it because oAuthScheme.scope will be never `null | undefined` at the point. right?